applications/portfolioDrafts/utils/validation.ts

```diff
@@ -15,6 +16,30 @@ export function isValidJson(str: string): boolean {
   return true;
 }
 
+/**
+ * Check whether a given string body is empty
+ *
+ * @param body - The body of the request
+ * @returns true if the string is empty or null
+ */
+export function isBodyPresent(body: string | null): body is string {
```

Comment:
This is may be the wrong use of a type guard. It does accomplish the goals we have and it does in fact assert that the input is a string. But it fails to capture the additional restrictions imposed within the body.